packages/amplify-cli/src/commands/configure.ts

```diff
@@ -15,12 +16,25 @@ export const run = async (context: Context) => {
     context.print.success('Usage Data has been turned off');
     return;
   }
-  if (context.parameters.options['usage-data-on']) {
+
+  if (context.parameters.options['usage-data-on ']) {
     write(context, { usageDataConfig: { isUsageTrackingEnabled: true } });
     context.print.success('Usage Data has been turned on');
     return;
   }
 
+  if (context.parameters.options['share-project-config-off']) {
+    DebugConfig.Instance.setAndWriteShareProject(false);
+    context.print.success('Share Project Config has been turned off');
+    return;
+  }
+
+  if (context.parameters.options['share-project-config-on']) {
+    DebugConfig.Instance.setAndWriteShareProject(true);
+    context.print.success('Share Project Config has been turned on');
+    return;
+  }
+
```

Comment:
These lines are so spaghetti 😂  any suggestions how we can optimize it?

Reply:
at least it's not spaghetti on christmas tree like in other places ...